atspi-common/src/events/mod.rs

```diff
@@ -99,6 +135,18 @@ impl From<EventBodyQT> for EventBodyOwned {
 	}
 }
 
+impl Default for EventBodyOwned {
```

Comment:
Similar question: Why have you implemented `Default` manually here?

Reply:
Cannot derive if not all fields implement `Default` and [`Value`](https://docs.rs/zvariant/3.14.0/zvariant/enum.Value.html) hasn't.